### DISCUSSION TITLE
Latest Release API endpoint is not returning the latest release...

### DISCUSSION THREAD

**Discussion by nvahalik** [community/community#20974] [Other Feature Feedback, Questions, & Ideas]
I have an action that creates a release. The problem is that _sometimes_ the release doesn't show up as the latest release.

Here's what I'm seeing:

![CleanShot 2022-07-20 at 16 46 43@2x](https://user-images.githubusercontent.com/632120/180088026-17d5f5de-2fb9-48a9-899b-3884a40a89a5.png)

**Comment by Adamantcheese** [comment]
I submitted a ticket for this because it happened to me as well, and it's quite stupid what's actually going on. Basically, because you released two things in one day, the API ignores the time in the day that the releases were made and only uses the date. After that, it sorts based on the tag, using SemVer first and then alphanumerically. In your case, 3608274 is alphanumerically greater than 1d441ae, so it returns the first one, despite being actually an older release.

**Reply by aratz-lasa** [reply]
Moreover, SemVer is not correctly implemented since two releases in the same day `v0.0.0-alpha.10` < `v0.0.0-alpha.2`. The pre-release versions sorts them alphanumerically.
